components/Item/Item.js

```diff
@@ -1,11 +1,13 @@
 import React from "react";
-import { Text, TouchableOpacity } from "react-native";
+import { Text, TouchableOpacity, View } from "react-native";
 import { style } from "./Item.Style";
 
 export default function Item({ item }) {
   return (
     <TouchableOpacity style={[style.item]}>
-      <Text style={[style.name]}>{item.name}</Text>
+      <View>
```

Comment:
I'm confused why the View is here? does it serve a purpose?

Reply:
So, my the idea to add a View inside Item was to use it as a container for the Item component for its own default styles